packages/instant/src/util/big_number.ts

```diff
@@ -0,0 +1,29 @@
+import { BigNumber } from '@0x/utils';
+import * as _ from 'lodash';
+
+/**
+ *  A BigNumber extension that is more flexible about decimal strings.
+ *  Such as allowing:
+ *  new BigNumberInput(0.) => 0
+ *  new BigNumberInput(1.) => 1
+ *  new BigNumberInput(1..) => still throws
+ */
+export class BigNumberInput extends BigNumber {
```

Comment:
I feel like the class name should match the file name

Reply:
extending BigNumber makes me a little nervous, but after thinking about it I understand why you went with this approach, and I can't immediately think of a better one

Reply:
This is pretty safe IMHO